packages/twenty-server/test/integration/graphql/suites/search/search-resolver.integration-spec.ts

```diff
@@ -55,8 +55,7 @@ describe('SearchResolver', () => {
         },
       });
 
-      const listingObjectMetadata = objectsMetadata.find(
-        // @ts-expect-error legacy noImplicitAny
+      const listingObjectMetadata = objectsMetadata.objects.find(
```

Comment:
objectsMetadata looks badlyname: objectsMetadataResponse ?

Reply:
Tbh just fixed a type issue due to the centralized tool return type mutation